README.md

```diff
@@ -87,13 +87,18 @@ polymer-bundler as a library has two exported function.
 
 `polymer-bundler` constructor takes an object of options similar to the command line options:
 
-- `excludes`: URLs to exclude from inlining. URLs may represent files or folders. HTML tags referencing excluded URLs are preserved.
-- `inlineCss`: Inline external stylesheets.
-- `inlineScripts`: Inline external scripts.
-- `sourcemaps`: Honor (or create) sourcemaps for inline scripts
-- `stripComments`: Remove non-license HTML comments.
+- `analyzer`: An instance of `Analyzer` (from `polymer-analyzer` package) which provides analysis of and access to files to operate on.
+- `excludes`: Array of URLs to exclude from inlining. URLs may represent files or folders. HTML tags referencing excluded URLs are preserved.
+- `inlineCss`: Inlines content of external stylesheets, i.e. `<link rel="stylesheet" href="...">`, into `<style>` tags in the bundled html.  Defaults to `true`.
+- `inlineScripts`: Inline content of external scripts, i.e. `<script src="...">`, into `<script>` tags in the bundled html.  Defaults to `true`.
+- `rewriteUrlsInTemplates`: Fix URLs found inside element attributes `action`, `assetpath`, `href`, `src`, `style` for elements inside `<template>` tags.  Defaults to `false`.
+- `sourcemaps`: Honor (or create) sourcemaps for inline scripts.  Defaults to `false`.
+- `stripComments`: Remove non-license HTML comments.  Defaults to `false`.
 
-`.generateManifest()` takes a collection of entrypoint urls and promises a `BundleManifest` which describes all the bundles it will produce.
+`.generateManifest()` takes a collection of entrypoint urls and promises a `BundleManifest` which describes all the bundles it will produce.  This method also takes two optional arguments:
```

Comment:
nit: I like `promises` but I'm not sure if that's commonly understood phrasing. 

Suggestion: "returns a promise that resolves with a `BundleManifest` description of all bundles it will produce"